tests/e2e/scenarios/test_reborn_webui_v2_legacy_extensions.py

```
@@ -713,6 +713,31 @@ async def test_reborn_legacy_extensions_offline_attempts_catalog_requests(
     page = await context.new_page()
     catalog_requests: list[str] = []
 
+    async def handle_llm_providers(route):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `handle_llm_providers`

Add return type annotation: `None`

(ANN202)